src/templates/_macros/app_tile.html

```diff
@@ -12,6 +12,7 @@
       src -- if tile is a link, attach a src param for analytics purposes
   #}
   {% set app = apps.transform(app) %}
+  {% set name = app.name if is_detail else app.short_name %}
```

Comment:
I don't think it would be measurably less performant. This is getting rendered in a loop (or once) that will fire at most about 20 times. If we're worried about an `if`/`else` in our templates maybe we should consider a different templating engine.

Using `short_name` on the listing page and `name` on the detail page has nothing to do with updating websites and apps to have the same interface which is what `apps.transform` should accomplish.